src/lib/services/api-token-service.ts

```diff
@@ -54,6 +59,30 @@ export class ApiTokenService {
         return this.store.getAllActive();
     }
 
+    private async initApiTokens(tokens: string[]) {
+        const tokenCount = await this.store.count();
+        if (tokenCount) {
+            return;
+        }
+        try {
+            for (const token of tokens) {
+                const tokenParts = token.split(':');
+                const newToken: IApiToken = {
+                    createdAt: undefined,
+                    project: tokenParts[0],
+                    environment: tokenParts[1],
+                    secret: `${tokenParts[0]}:${tokenParts[1]}.${tokenParts[2]}`,
+                    type: ApiTokenType.ADMIN,
```

Comment:
Are we sure it only makes sense to only support Admin tokens as part of init? I am leaning towards yes, because you can use these to create client SDK tokens. 

What do you think @chriswk?

Reply:
It seems reasonable, as you say, that will allow you to use the created admin token to create necessary client tokens.